jcoin.datacommons.io/portal/gitops.json

```diff
@@ -492,16 +492,20 @@
           {
             "title": "Constructs",
             "fields": [
-              "cmc_data_type",
+              "cmc_subject",
```

Comment:
Using `cmc_subject` in the portal config at the same time as adding it to the ETL mapping means the explorer page will be broken until the next time the ETL runs, and will stay broken if for some reason the ETL fails. We can do it if the page being down for a bit is not a problem.

Reply:
It'll break in QA but not PROD because [gitops.json for PROD has cmc_subject](https://github.com/uc-cdis/cdis-manifest/blob/b4fe17d55a069da2f9f3b67db8df8e11d4dd602e/jcoin.datacommons.io/portal/gitops.json#L493). I'll change the QA to correct version

Reply:
@paulineribeyre  ok I deleted cmc_data_type in QA and it is now consistent

Reply:
mm, i'm confused, the changes listed in the PR don't seem accurate. The "files changed" tab shows changes that are already on master. Maybe a rebase would help?

Reply:
sorry for the confusion and appreciate the tip!  I think a rebase worked as its now showing only the manifest DD change.

Reply:
ah it makes a lot more sense now! 😄

Reply:
good deal - lets see..anything else before a merge/deployment?

Reply:
waiting for the tests results